### PR TITLE
PAT generation via ADAL with username password

### PR DESCRIPTION
**Pull request by mitchdenny**

### CODE REVIEW

File: NonInteractivePatGenerationSample/NonInteractivePatGenerationSample/Program.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        // This is the resource ID for the VSTS application - don't change this.
+        private const string VstsResourceId = "499b84ac-1321-427f-aa17-267ca6975798";
+
+        // This is the resource for your own application registered in AAD (which needs access to the VSTS application).
+        private const string ClientId = "[your AAD application client ID]";
+
+        public static void Main(string[] args)
+        {
+            var adalCredential = new UserPasswordCredential(
+                "[your AAD username]", //
```
Comment (justmarks) [inline NonInteractivePatGenerationSample/NonInteractivePatGenerationSample/Program.cs]: why the empty comment?

**Comment by mitchdenny** [comment]
@justmarks I added a README etc.

**Comment by mitchdenny** [comment]
Re-opening to merge ;P

**Comment by peakyy** [comment]
I would actually walk through what we are looking for for each of the [bracketed user options]. This will keep the read me consistent with other samples in this repo and also it gives chance to clarify that the "application Id" given by AAD is the "client id" we are asking them to fill in. I want to ensure there's no confusion over the different names we are using.

**Pull request merged**